server/session/session.go

```diff
@@ -56,7 +56,13 @@ func (s *Server) Delete(ctx context.Context, q *session.SessionDeleteRequest) (*
 // chicken-and-egg situation if we didn't place this here to allow traffic to pass through.
 func (s *Server) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
 	// this authenticates the user, but ignores any error, so that we have claims populated
-	ctx, _ = s.authenticator.Authenticate(ctx)
+	// We only ignore errors for Create method
+	ctx, err := s.authenticator.Authenticate(ctx)
+	if fullMethodName != "/session.SessionService/Create" {
```

Comment:
We just really only have to have `/session.SessionService/Create` method as the only unprotected method in this API, since this the one issuing the client an auth token.